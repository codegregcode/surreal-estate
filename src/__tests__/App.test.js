import React from "react";
import { render, screen } from "@testing-library/react";
import App from "../components/App";

describe("App", () => {
  const { asFragment } = render(<App />);

  it("render App component", () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
