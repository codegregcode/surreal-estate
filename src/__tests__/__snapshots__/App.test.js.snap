// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render App component 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <img
        alt="logo"
        src="logo.png"
      />
      <ul
        class="navbar-links"
      >
        <li
          class="navbar-links-item"
        >
          View Properties
        </li>
        <li
          class="navbar-links-item"
        >
          Add a Property
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
